src/allmydata/test/web/test_web.py

```diff
@@ -8,32 +8,20 @@
 
 from twisted.application import service
 from twisted.internet import defer
-from twisted.internet.defer import inlineCallbacks, returnValue, maybeDeferred
+from twisted.internet.defer import inlineCallbacks, returnValue
 from twisted.internet.task import Clock
 from twisted.web import client, error, http
 from twisted.python import failure, log
 
-from nevow.context import WebContext
-from nevow.inevow import (
-    ICanHandleException,
-    IRequest,
-    IData,
-)
-from nevow.util import escapeToXML
-from nevow.loaders import stan
-from nevow.testutil import FakeRequest
-from nevow.appserver import (
-    processingFailed,
-    DefaultExceptionHandler,
-)
+from twisted.words.xish.domish import escapeToXml
```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaa let's see if we can find a better API than this (`twisted.words.xish.domish` is not one of the more battle-tested areas of Twisted and "xml parsing and serialization" is not exactly Twisted's forte).

Reply:
Would `twisted.web.microdom.escape()` work better?  

It doesn't seem that Tahoe-LAFS uses any specialized XML library directly or indirectly, and I don't know if adding one just for this small use case would be worth the hassle.

Reply:
Maybe we can go in a different direction.  We're not actually emitting xml here anyway, but html!  Perhaps the stdlib `html.escape`?

Reply:
Good idea, I did not even care to think what is actually being done here.  Thank you!

Reply:
There's a small difference in behavior:

Here's a string, called `s`: `<&weirdly'named"file>>>_<iframe />.txt`

Result of `escapeToXml(s)` is `&lt;&amp;weirdly'named"file&gt;&gt;&gt;_&lt;iframe /&gt;.txt`.
Result of `html.escape(s)` is `&lt;&amp;weirdly&#x27;named&quot;file&gt;&gt;&gt;_&lt;iframe /&gt;.txt`.

That single quote has been escaped to `&#x27` by `html.escape()`, so a test case had to be changed accordingly.  I don't know if that is a good thing or bad thing.

Reply:
Also replaced `cgi.escape()` with `html.escape()`, because they look identical to me, and because `cgi.escape()` seems to have disappeared in newer Pythons.

Reply:
I think that is all correct and reasonable.